linux_os/guide/system/software/updating/dnf-automatic_security_updates_only/oval/shared.xml

```diff
@@ -0,0 +1,14 @@
+{{{
+oval_check_config_file(
```

Comment:
Please, consider creating a macro for this case as it is very similar to: https://github.com/ComplianceAsCode/content/pull/4613/files#diff-dcd0373f534bb36b76e776f6d3b44df8R2

You can even introduce a new parameter for the [`commands`] section in the configuration file.

Reply:
It would be great to have a macro specifically for .ini files

Reply:
@ggbecker I would like to create a new macro, in a separate PR.

Reply:
> It would be great to have a macro specifically for .ini files

Agreed.

Reply:
See this PR https://github.com/ComplianceAsCode/content/pull/4626 which introduces this macro